open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/auditlog/OMAGAdminAuditCode.java

```diff
@@ -86,89 +86,174 @@
+                    " The error message is {2}",
+            "The admin services detected an error during the start up of a specific access service subsystem.  " +
+                    "Its services are not available for the server and so the server failed to start.  Details of the error " +
+                    "that the access service detected are logged and a detailed exception is returned to the caller. ",
+            "Review the error message and the other failures reported by the access service to determine the cause of the problem.  It is " +
+                    "typically either " +
+                    "a configuration error, or one of the external resources it needs, such as a topic on the event bus, is not  " +
+                    "available. Access services may be developed by the Egeria community or a third party.  If the error messages " +
+                    "are not sufficient to resolve the problem, raise an issue with the author of the access service to get this " +
+                    "improved.  Once the root cause of the error is resolved, restart the server."),
 
     NULL_ACCESS_SERVICE_ADMIN_CLASS("OMAG-ADMIN-400-011 ",
-                                    OMRSAuditLogRecordSeverity.EXCEPTION,
-                                    "The OMAG server {0} has been passed a null admin services class name for access service {1}",
-                                    "The system is unable to initialize this access service. The server failed to start.",
-                                    "If the access service should be initialized then set up the appropriate admin services class name " +
-                                            "in the access service's configuration and restart the server instance. Otherwise, " +
-                                            "remove the configuration for this access service and restart the server."),
+            OMRSAuditLogRecordSeverity.EXCEPTION,
+            "The OMAG server {0} has been passed a null admin services class name for access service {1}",
+            "The system is unable to initialize this access service. The server failed to start.",
+            "If the access service should be initialized then set up the appropriate admin services class name " +
+                    "in the access service's configuration and restart the server instance. Otherwise, " +
+                    "remove the configuration for this access service and restart the server."),
+
+    STARTING_VIEW_SERVICES("OMAG-ADMIN-0020",
+            OMRSAuditLogRecordSeverity.STARTUP,
+            "The Open Metadata View Services (OMVSs) are starting",
+            "The operational admin services are initializing the access service subsystems in a metadata server instance.  " +
+                    "These provide specialist task orientated APIs for viewing open metadata.  The view services support " +
```

Comment:
These are view services not access services and they run in a View server not a Metadata server.